hack/test-stirunimage.sh

```diff
@@ -86,6 +86,20 @@ test_debug "s2i build with relative path with file://"
 s2i build file://./cakephp-ex openshift/php-55-centos7 test --loglevel=5 &> "${WORK_DIR}/s2i-rel-proto.log"
 check_result $? "${WORK_DIR}/s2i-rel-proto.log"
 
+if [[ "$OSTYPE" == "cygwin" ]]; then
```

Comment:
I think that if you use $S2I_WORK_DIR instead of $WORK_DIR, you should not need this if/then/else.
On non-Windows systems, S2I_WORK_DIR equals WORK_DIR.  On Windows systems, S2I_WORK_DIR translates foo/bar to C:\path\to\foo\bar for you.